server/README.md

```diff
@@ -8,7 +8,7 @@ If the refresh token expires (after 1 day for security reasons), the client need
 
 ### Running the server
 
-1. Create a virtual environment and install the packages: `virtualenv venv && source venv/bin/activate && pip install  -r requirements.txt`.
+1. Create a virtual environment and install the packages: `python3 -m venv venv && source venv/bin/activate && pip install -U pip && pip install  -r requirements.txt`.
```

Comment:
I would argue that python 2 is not maintained anymore since January 1 2020. Django 3 doesn't support python 2 anymore and it seems to me that it is now more typical for people to use python 3 with Django, but I can be wrong.
I'll leave it if you prefer.

Reply:
Sorry, I need to work on my communication skills and generally not confusing people:

> Not very typical for people to use python3

We're totally on the same page when you say:

> I would argue that python 2

I meant not very typical for people to create a virtualenv command with `python3 -m venv venv`.

Just keep it simple. I'd much rather you regressed this change.

Reply:
no problem